release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1335,7 +1368,9 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *OLM*
 
+* Previously, `CustomResourceDefinition` (CRD) objects applied as part of an Operator installation could sometimes satisfy the installation requirements of a newer version of the same Operator. Consequently, during an Operator upgrade, the version being replace could be prematurely removed. In some cases, the upgrade would stop. With this update, the CRDs that are created or updated as part of the Operator bundle installation are annotated to indicate their bundle of origin. These annotations are used by the `ClusterServiceVersion` (CSV) controller to distinguish between pre-existing CRDs and same-bundle CRDs. As a result, upgrades will not complete until the current version's CRDs have been applied. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1947946[*BZ#1947946*])
```

Comment:
s/being replace/being replaced

I don'e see _`ClusterServiceVersion` (CSV) controller_ in the docs. Is this correct? cluster Should be _service version (CSV) controller_ ??